### ISSUE TITLE
AKS offer: not able to select Node Size in WestUS

### ISSUE THREAD

**Issue by galiacheng** [oracle/weblogic-azure#292]
Description: Not able to select Node size from Azure Portal when selecting region **WestUS**.

Offer: WLS on AKS
Reproduce steps:
- Open https://portal.azure.com/#create/oracle.20210620-wls-on-aks20210620-wls-on-aks from Azure portal.
- Input resource group name and select region **WestUS** in Basics blade
- Open **AKS** blade, the node size selection control is gray and not able to select size. See the screenshot.

![image](https://github.com/oracle/weblogic-azure/assets/59823457/c70faee1-7206-4d15-9208-f62917d63033)

Expected result: Node size should be able to select in **WestUS**.

**Comment by galiacheng** [comment]
Error observed in browser developer tools: 

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'zones')
    at BRszf-QjJduq.js:189:3328
    at Array.filter (<anonymous>)
    at BRszf-QjJduq.js:189:3172
```

**Comment by galiacheng** [comment]
Asked in https://stackoverflow.microsoft.com/questions/386260

**Comment by galiacheng** [comment]
Root cause: the recommendedSize "Standard_DS2_v2" is not available in West US.

**Comment by galiacheng** [comment]
WestUS is a region where no zones are defined for any of the disk/virtualmachine resources, this is the reason why the control is "disabled". The current scenario should result in an error state. 

UI team has fixed the issue in [Pull request 9467718: Size Selector BugFix - Repos (visualstudio.com)](https://msazure.visualstudio.com/One/_git/AzureUX-CreateUIDef/pullrequest/9467718).

The fix is live, close this issue.

![image](https://github.com/oracle/weblogic-azure/assets/59823457/b3ca99f8-8f3c-43a7-85ef-03607a87a565)
